app/models/node.rb

```diff
@@ -851,6 +851,14 @@ def can_tag(tagname, user, errors = false)
       errors ? I18n.t('node.only_admins_can_lock') : false
     elsif tagname.split(':')[0] == 'redirect' && Node.where(slug: tagname.split(':')[1]).length <= 0
       errors ? I18n.t('node.page_does_not_exist') : false
+    elsif tagname.split(':')[0] == 'facebook'
+      errors ? "Only Oauth can create such tags" : false
+    elsif tagname.split(':')[0] == 'twitter'
+      errors ? "Only Oauth can create such tags" : false
+    elsif tagname.split(':')[0] == 'google'
+      errors ? "Only Oauth can create such tags" : false
+    elsif tagname.split(':')[0] == 'github'
+      errors ? "Only Oauth can create such tags" : false
```

Comment:
By using `["facebook", ...].include? tagname.split(':')[0]` instead we can squeeze 8 lines into just 2 and avoid much repetition. What do you think?

Reply:
@seafr I want to know whether we want the same error message for all 4 cases or different for these. If same message is OK. I will do the required patch.
Thanks

Reply:
Ok, I see your point. I thought this was the final version as the `in progress` label was removed.

Reply:
@jywarren what do you suggest?

Reply:
Hm, i like @seafr's suggestion! Let's do it!

Reply:
@SidharthBansal I suggest moving all 4 options into 1 `elseif` as I said above. You can also customise the error message to something like: 
`"... create the #{tagname...[0]} tag"` 
if you want to make it different for each case.

Reply:
Ok, updating in a minute. Thanks both of you